sdk/src/main/java/io/dapr/client/domain/BulkPublishResponse.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Class representing the response returned on bulk publishing events.
+ */
+public class BulkPublishResponse {
+
+  private String errorCode;
```

Comment:
Do you think we should remove this field since we have changed the response here: https://github.com/dapr/dapr/blob/master/dapr/proto/runtime/v1/dapr.proto#L336 ?

Reply:
Morever, I don't think we use that field at all: https://github.com/dapr/java-sdk/blob/48b3a6c8d6dad74bdd90edc25f7d5e168a5c1538/sdk/src/main/java/io/dapr/client/DaprClientGrpc.java#L258